src/components/editor/Editor.tsx

```diff
@@ -58,9 +62,15 @@ const Editor: React.FC<{}> = () => {
           download={structureDefinition.name + ".json"}
         >
           <button>Download</button>
-        </a>
-        {renderAttributeEditor(structureDefinition)}
-        <ResourceProfileMapping profile={structureDefinition} />
+        </a>*/}
+        <div className={classes.mapping}>
+          <Paper className={`${classes.paperLeft} ${classes.paper}`}>
```

Comment:
If you would like to inject several class names in a more elegant way, try having a look at the 'clsx' method.
It also allows you to conditionally set class names

```suggestion
          <Paper className={clsx(classes.paperLeft, classes.paper, {
               [classes.conditionalClassname]: condition
          })}>
```